### DISCUSSION TITLE
Issue Running Ionic App on Real iOS Device with Capacitor (Version Information Included)

### DISCUSSION THREAD

**Discussion by Nikita199108** [ionic-team/capacitor#6880] [General]
Hello,

I am currently learning and encountering an issue when attempting to run my Ionic app on a real iOS device using Capacitor. The problem occurs when I use the 'ionic capacitor run ios' command. After providing my login credentials, the app starts loading but then hangs, and there is no further progress. This issue does not manifest when I use the 'ionic capacitor run ios -l' command for live reloading in development mode.

Here is the relevant version information of the Capacitor dependencies I am using:

"@capacitor/app": "^4.1.0"
"@capacitor/core": "^5.2.2"
"@capacitor/ios": "^5.2.2"
I have taken several steps to troubleshoot this issue, including checking my Xcode configuration, verifying the iOS device's developer mode, and ensuring my Capacitor configuration is correct.

Despite these efforts, the problem persists. I would appreciate any guidance or insights on how to resolve this issue and successfully run my Ionic app on a real iOS device using Capacitor.

Thank you for your assistance.

**Comment by jcesarmobile** [comment]
First update `@capacitor/app` plugin to latest 5.x version available, capacitor and capacitor plugin major versions should match

Then debug your app as explained here
https://ionicframework.com/docs/troubleshooting/debugging

Looks like some CORS problem or some API that doesn't work when using a custom scheme as Capacitor does.
